### PR TITLE
Add password protection for database

### PR DESCRIPTION
**Pull request by rochala**

### CODE REVIEW

File: deployment/local.conf [hunk 1]
```diff
@@ -6,6 +6,7 @@ com.olegych.scastie {
     remote-sbt-ports-start = 5150
     remote-sbt-ports-size = 1
 
+    snippets-container = files
```
Comment (julienrf) [inline deployment/local.conf:9]: Not very important, but “container” made me think about Docker. Maybe “storage” would be more self-explanatory?

File: project/Deployment.scala [hunk 2]
```diff
@@ -95,6 +95,8 @@ class ScastieConfig(val configurationFile: File) {
   val balancerConfig = config.getConfig("com.olegych.scastie.balancer")
   val runnersHostname = balancerConfig.getString("remote-hostname")
   val sbtRunnersPortsStart = balancerConfig.getInt("remote-sbt-ports-start")
+  val containerType = balancerConfig.getString("snippets-container")
```
Comment (julienrf) [inline project/Deployment.scala:98]: Where do we set the default value for this configuration key? I see it is set only in the test resources of the balancer module.

Comment (rochala) [reply project/Deployment.scala:98]: This key was already present in the codebase and its configuration is defined for staging, dev, local deployment and also dev / test dev.
```
balancer/src/main/resources/reference.conf
2:  snippets-container = files

balancer/src/test/resources/application.conf
2:  snippets-container = files

deployment/local.conf
9:    snippets-container = files

deployment/staging.conf
6:    snippets-container = mongo

deployment/production.conf
6:    snippets-container = mongo

project/Deployment.scala
98:  val containerType = balancerConfig.getString("snippets-container")

server/src/main/resources/reference.conf
4://    snippets-container = mongo
5:    snippets-container = files
```

**Review by julienrf — APPROVED** [review]
Thank you!

**Pull request merged**